components/collapsible-panel/collapsible-panel.js

```diff
@@ -374,6 +376,11 @@ class CollapsiblePanel extends SkeletonMixin(FocusMixin(RtlMixin(LitElement))) {
 
 	_handleSummarySlotChange(e) {
 		const content = e.target.assignedNodes({ flatten: true });
+		for (const element of content) {
+			if (this.skeleton) element.setAttribute('skeleton', true);
+			else element.removeAttribute('skeleton');
+		}
```

Comment:
Do we want the panel to be auto assigning skeleton to the summary items? What we have done for other components with slots like lists, is give the responsibility of adding skeleton to the slot content to the consumer. That would also give the consumer the ability to separately assign skeleton to the different summary items if it was needed.

In this case though, the slot only accepts one type of component that is tightly coupled with the panel data so maybe there is a case for this. Interested to hear if others think differently.

Reply:
Yeah that was my thought process too. The only problem is we are not actually locking down consumers from adding other components so I was thinking we could either impose that restraint or set skeleton automatically but only for summary items.

Reply:
The use of `li` in `d2l-collapsible-panel` is in an unused FACE related repo, they are still using `li` in `d2l-labs-accordion` but we can change that when backporting.

Reply:
We should probably either lock the slot down first or make sure we're only setting the skeleton on summary items similar to what we are doing in [filter](https://github.com/BrightspaceUI/core/blob/4b95ca8e8fb4c3406bce07cf7e61b088ef8dc5d2/components/filter/filter.js#L565).

Also, I think you should be able to use the JS property to set the `skeleton`:
e.g. `element.skeleton = true`

Reply:
I spoke with @geurts and we will set skeleton only on the summary items slotted, other components are left for the consumers to handle.

Reply:
I am late to reading this thread, but I don't think we should do this. The light-DOM belongs to the consumer, so I think it should be up to them to set the skeleton attribute. In general I don't think we should be updating DOM outside our scope of ownership... though I know we do from time to time.